### DISCUSSION TITLE
Qiita APIのStockについて

### DISCUSSION THREAD

**Discussion by leng-z4** [increments/qiita-discussions#112] [ヘルプ]
Qiita APIにおいて、カテゴリーを指定して自分がStockした記事を取得したり、新しく記事をStockしたりする方法はあるのですか。カテゴリーを指定する方法が分かりません。

**Comment by kskwtnk** [comment]
@Zero-484 

ご意見ありがとうございます！
そして、申し訳ありません……現状ではカテゴリーを取得する方法がないのと、他の施策との兼ね合いがありAPIのアップデートがすぐには実施できなさそうです…… 🙇 

ちなみにもしよろしければ、カテゴリーをAPIで取得できた場合どのように使用するイメージでしたか？
教えていただけますと非常に嬉しいです！

**Reply by leng-z4** [reply]
取得したカテゴリー別に記事をストックしようと思っていました。

**Reply by kskwtnk** [reply]
@Zero-484 

ありがとうございます！
APIのアップデートにあたって参考にさせていただきます！

繰り返しになってしまいますが、すぐには実施できなさそうで、すみません 🙇
